templates/archives/stories/mayowa_story.jinja2

```diff
@@ -26,15 +26,29 @@ Mayowa Songonuga
     <figcaption class="figure-caption">Map of MIT in 1905. The Lowell building (then home to Course 6) is circled.</figcaption>
 </figure>
 
-<p>The first high-speed computer was installed at MIT in early 1957. Before that, MIT had other,
-    less advanced machinery that was used in the classroom setting. Computer Science at MIT started
-    as just a few classes, notably 6.535 and 6.536, Coding for Digital Machines and Coding for Analog
-    Machines. By June of 1954, automatic computation, as it was called then, had spread to many different
-    departments, including Mechanical, Chemical, and Aeronautical Engineering, Physics, and Mathematics.
-    These departments all offered classes that utilized automatic computing in some way, but there were
-    also eleven classes directly related to the emerging field. Of these eleven, ten were graduate electives,
-    and the last was a senior elective. 6.535 evolved to  Introduction to Computer Coding and Logic and
-    6.536 to Machine Computation. Other classes included M411, Numerical Analysis, and 6.568, Switching Circuits.</p>
+<p>The first high-speed computer was installed at MIT in early 1957. Before that, MIT had other, less advanced machinery
+    that was used in the classroom setting. Computer Science at MIT started as just a few classes, notably 6.535 and
+    6.536, Coding for digital Machines and Coding for Analog Machines. By June of 1954, automatic computation, as it
+    was called then, had spread to many different departments, including Mechanical, Chemical, and Aeronautical
+    Engineering, Physic, and Mathematics. These departments all offered classes that utilized automatic computing in
+    some way, but there were also eleven classes directly related to the emerging field. Of these eleven, ten were
+    graduate electives, and the last was a senior elective. 6.535 evolved to  Introduction to Computer Coding and Logic
+    and 6.536 to Machine Computation. Other classes included M411, Numerical Analysis, and 6.568, Switching Circuits. </p>
+
+<p>Currently, there are over 300 classes offered by the course 6 departments alone, not including all of the classes
```

Comment:
People outside of MIT won't know what `course 6` means, unless you introduce the term earlier. Otherwise you can write out the name.